jetty-ee9/jetty-ee9-servlet/src/test/resources/contextResources/subdir/index.html

```diff
@@ -0,0 +1,3 @@
+<html>
+ <body> <h1>WELCOME</h1></body>
+</html>
```

Comment:
end of line missing

Reply:
Is this file actually used?

Reply:
Yes, the file has to exist for the redirect to be sent.